proc.c

```diff
@@ -683,20 +683,23 @@ rb_block_clear_env_self(VALUE proc)
  *  <i>params</i> using something close to method calling semantics.
  *  Generates a warning if multiple values are passed to a proc that
  *  expects just one (previously this silently converted the parameters
- *  to an array).  Note that prc.() invokes prc.call() with the parameters
- *  given.  It's a syntax sugar to hide "call".
+ *  to an array).  Note that <code>prc.()</code> invokes
+ *  <code>prc.call()</code> with the parameters given.  It's a syntax sugar to
```

Comment:
"Syntactic sugar" is correct. "Syntax sugar" is not.

Seriously - just [google it](https://www.google.co.uk/search?q="syntax+sugar")! Not only does "syntactic sugar" yield 10x more results, but most of the results for "syntax sugar" are basically just redirects, e.g. on [wikipedia](http://en.wikipedia.org/w/index.php?title=Syntax_sugar&redirect=no).

Simply comparing the popularity of the words "syntax" and "syntactic" is irrelevant here - that would be like claiming it's better to write "swim pool" than "swimming pool", because "swim" is a more common word than "swimming"!!